CODE/VulnScan/tools/_vectorizer.py

```
@@ -0,0 +1,53 @@
+
+def choose_vectorizer(vectorizer_types: str) -> TfidfVectorizer | CountVectorizer:
+    print("Vectorizer Type: ", vectorizer_types)
+    print("Vectorizing Data...")
+    if vectorizer_types == 'tfidf':
+        return TfidfVectorizer(max_features=10000)
+    if vectorizer_types == 'count':
+        return CountVectorizer(max_features=10000)
+    raise ValueError("Unsupported vectorizer type. Choose 'tfidf' or 'count'.")
+
+
+def main(data_paths: str, vectorizer_types: str, output_paths: str):
```

Ruff (0.8.0)

Missing return type annotation for private function `main`

Add return type annotation: `None`

(ANN202)